src/server/serverSetup.js

```diff
@@ -252,3 +334,11 @@ sjrk.storyTelling.server.getThemePath = function (theme, themeFolder) {
 
     return themePath;
 };
+
+sjrk.storyTelling.server.makeMemorySessionStore = function () {
+    // TODO: Currently using https://www.npmjs.com/package/memorystore as it is a production ready memory store;
```

Comment:
If this `TODO` is not intended to be tackled in SJRK-405, please file a Jira to do that work and add a link to that Jira here

Reply:
https://issues.fluidproject.org/browse/SJRK-444